Implementing CutFlowManager.

jniedzie opened the pull request.

jniedzie committed:
Implementing CutFlowManager.

jniedzie committed:
Moving TTAlps specific cuts to a separate class.

kerstinlovisa commented on libs/extensions/src/EventProcessor.cpp:
Do we want to be able to use the EventProcessor without "triggerSelection" in the config? When we do a second round of skimming for example?

jniedzie replied:
Absolutely, you're right - thanks :)
I pushed a solution involving exceptions - should cover the case you spotted.

jniedzie committed:
Adding a general exception class + handling a case
where trigger selections are not required and therefore not definde
in the config file.
